doc/weights.xml

```diff
@@ -219,3 +219,51 @@ fail]]></Example>
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="DigraphMaximumFlow">
+<ManSection>
+  <Attr Name="DigraphMaximumFlow" Arg="digraph, start, destination"/>
+  <Returns>A record.</Returns>
+  <Description>
+    If <A>digraph</A> is an edge-weighted digraph with vertices <A>start</A> and
+    <A>destination</A>, this returns a record representing the maximum flow from
+    <A>start</A> to <A>destination</A> in the digraph.<P/>
+
+    A <E>flow</E> is a function from the edges of <A>digraph</A> to the positive
+    real numbers, such that:
+    <List>
+      <Item>
+        Each edge's flow is no more than its weight;
+      </Item>
+      <Item>
+        For each vertex other than <A>start</A> and <A>destination</A>, the sum
+        of flows for all incoming edges is equal to the sum of flows for all
+        outgoing edges;
+      </Item>
+      <Item>
+        The sum of flows of edges leaving <A>start</A> is equal to the sum of
+        flows of edges entering <A>destination</A> (this sum is called
+        <M>M</M>).
+      </Item>
+    </List>
+    A <E>maximum flow</E> is a flow that maximises the value of <M>M</M>.<P/>
+
+    The flow is represented as a list of lists where each entry is a number
+    representing the flow on the edge in the corresponding position in
+    <C>OutNeighbours(<A>digraph</A>)</C>.<P/>
+
+    Note that the size <M>M</M> of the flow can be found with
+    <C>Sum(DigraphMaximumFlow(<A>digraph</A>, <A>start</A>,
+    <A>destination</A>)[<A>start</A>])</C>. <P/>
+
+    See <Ref Attr="EdgeWeights" Func="EdgeWeightedDigraph"/>.
+    <Example><![CDATA[
```

Comment:
Be good to have a note of the complexity, to understand what sort of computations are feasible with this implementation.